### PR TITLE
Playdigo: new adapter

### PR DESCRIPTION
**Pull request by Yanivplaydigo**
doc - https://github.com/prebid/prebid.github.io/pull/5274

### CODE REVIEW

File: adapters/playdigo/playdigo.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+		adapterReq, err := a.makeRequest(&reqCopy)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+
+		if adapterReq != nil {
+			adapterRequests = append(adapterRequests, adapterReq)
+		}
```
Comment (gargcreation1992) [inline adapters/playdigo/playdigo.go:82]: you should remove this check. This is not required here.

Comment (gargcreation1992) [inline adapters/playdigo/playdigo.go:82]: you could directly say - `adapterRequests = append(adapterRequests, adapterReq)`

File: adapters/playdigo/playdigo.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
+	if len(response.Cur) != 0 {
+		bidResponse.Currency = response.Cur
+	}
+
+	for _, seatBid := range response.SeatBid {
+		for i := range seatBid.Bid {
+			bidType, err := getBidMediaType(&seatBid.Bid[i])
```
Comment (gargcreation1992) [inline adapters/playdigo/playdigo.go:132]: Requesting you to please use openrtb2.Bid.MType to define bid type. Example - https://github.com/prebid/prebid-server/blob/2cc2c90a09708de6fe5cdbf5d798a08db0b806d9/adapters/adelement/adelement.go#L124

File: static/bidder-info/playdigo.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+endpoint: "https://server.playdigo.com/pserver"
+geoscope:
+  - USA
+maintainer:
+  email: "yr@playdigo.com"
```
Comment (gargcreation1992) [inline static/bidder-info/playdigo.yaml:5]: sent an email for verification.

File: static/bidder-info/playdigo.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+endpoint: "https://server.playdigo.com/pserver"
```
Comment (gargcreation1992) [inline static/bidder-info/playdigo.yaml:1]: host is reachable - 
```
ag476c@vpn-10-249-108-83 bin % curl -v https://server.playdigo.com/pserver
*   Trying 80.77.86.11:443...
* Connected to server.playdigo.com (80.77.86.11) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Change cipher spec (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / AES128-SHA
* ALPN: server accepted http/1.1
* Server certificate:
*  subject: CN=*.playdigo.com
*  start date: Mar 18 09:19:05 2024 GMT
*  expire date: Jun 16 09:19:04 2024 GMT
*  subjectAltName: host "server.playdigo.com" matched cert's "*.playdigo.com"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* using HTTP/1.1
> GET /pserver HTTP/1.1
> Host: server.playdigo.com
> User-Agent: curl/8.4.0
> Accept: */*
>
< HTTP/1.1 204 No Content
< Server: nginx
< Date: Thu, 25 Apr 2024 09:26:40 GMT
< Connection: keep-alive
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Credentials: true
<
* Connection #0 to host server.playdigo.com left intact
* ```

File: static/bidder-info/playdigo.yaml (hunk 3 above)
Comment (gargcreation1992) [reply static/bidder-info/playdigo.yaml:5]: email verified.

File: adapters/playdigo/playdigo.go (hunk 2 above)
Comment (Yanivplaydigo) [reply adapters/playdigo/playdigo.go]: updated

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**